package on2015_11.on2015_11_01_Grand_Prix_of_Yekaterinburg__Div_2.B0;

import net.egork.chelper.tester.NewTester;

import org.junit.Assert;
import org.junit.Test;

public class Main {
	@Test
	public void test() throws Exception {
		if (!NewTester.test("lib/test/on2015_11/on2015_11_01_Grand_Prix_of_Yekaterinburg__Div_2/B0/B.task"))
			Assert.fail();
	}
}
